src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,446 @@
+then time information as detailed below should be added to the `*_recording-autosampler_blood.tsv`
+and the values below should be added to the `*_recording-autosampler_blood.json`.
+
+| **Column name** | **Description**                                                              | **Units**                             |
+| --------------- | ---------------------------------------------------------------------------- | ------------------------------------- |
+| time            | Time in relation to `TimeZero` defined by the `*_pet.json`. (for example, 5) | Unit of time steps in seconds |
+
+### Radioactivity in plasma
+
+This section may be omitted if plasma measurements of radioactivity were not made.
+It contains information regarding sampled plasma data.
+If plasma measurements are available by manual sampling, they should be added to the `*_recording-blood_discrete.tsv`
+and the values below should be added to the `*_recording-blood_discrete.json`.
+If plasma measurements are available by an autosampler, they should be added to the `*_recording-blood_continous.tsv`
+and the values below should be added to the `*_recording-blood_continous.json`.
+
+| **Column name**      | **Description**         | **Units**                                         |
+| -------------------- | ----------------------- | ------------------------------------------------- |
+| plasma_radioactivity | Radioactivity in plasma | Unit of plasma radioactivity (for example, "kBq/mL") |
+
+### Metabolites
+
+This section may be omitted if metabolite measurements were not made.
+If metabolite measurements are available by manual sampling
+they should be added to the `*_recording-blood_discrete.tsv`
+and the values below should be added to the `*_recording-blood_discrete.json`.
+If metabolite measurements are available by an autosampler they should be added to the `*_recording-blood_continous.tsv`
+and the values below should be added to the `*_recording-blood_continous.json`.
+Hence, it may contain information regarding metabolite info, such as the following three column examples:
+
+| **Column name**            | **Description**                                                                   | **Units**                                                   |
+| -------------------------- | --------------------------------------------------------------------------------- | ----------------------------------------------------------- |
+| metabolite_parent_fraction | Parent fraction of the radiotracer (0-1)                                          | Unit of parent fraction (for example, "unitless")           |
+| metabolite_polar_fraction  | Polar metabolite fraction of the radiotracer (0-1)                                | Unit of polar metabolite fraction (for example, "unitless") |
+| hplc_recovery_fractions    | HPLC recovery fractions (the fraction of activity that gets loaded onto the HPLC) | Unit of recovery fractions (for example, "unitless")        |
```

Comment:
Please replace "unitless" with "arbitrary".

Reply:
These units are not arbitrary though.  The units are cancelled out, so that they are a fraction, and hence unitless.

Reply:
Something similar came up in the qMRI BEP as well. There are a number of data types in that BEP where the data are on a specific scale, but without any specific unit, such as percentages. We chose to use the term "arbitrary" for such situations (see #580). If you think that there's a meaningful distinction to be made between truly arbitrary units and fractions of elements with meaningful units, then I think it merits a separate issue since such a distinction will be broadly applicable across the specification. I believe that this will also require changes to the ["Units" section of "Common Principles"](https://bids-specification.readthedocs.io/en/latest/02-common-principles.html#units), where "arbitrary" is defined.

Reply:
@mathesong does the definition of arbitrary to mean unitless make sense to you after Taylor's explanation?

Reply:
This is an interesting point.  I still feel like there's a meaningful difference between the use of arbitrary in, say, BOLD signal and here. Here, we have SI units in both the numerator and denominator, and they happen to cancel out. But there is nothing arbitrary about the units of either measurement or their ratio.

I looked through the Units section of the common principles, and didn't find a good definition of arbitrary there.  Am I just missing it?

I googled around a little bit about found [this](http://media.iupac.org/divisions/VII/VII.C.1/C-NPU_Uppsala_081023_25_minutes_confirmed.pdf) in item 19: these are minutes from IUPAC. They describe how they define "arbitrary units", where the measurements themselves are a property of, for example, the local laboratory in which the measurements are made.  To avoid ambiguity, they instead refer to them as "procedure defined units".

Importantly, I would argue that these parent fractions are *not* a property of a local laboratory, and not procedurally defined either.  They ought to be directly comparable between different laboratories.  Does this make sense?  Or does it come across like a trivial distinction?

Reply:
@tsalo : Is it necessary to resolve the arbitrary vs unitless distinction for the broader BIDS specification now before the commenting period runs out for PET BIDS?

To summarise from my previous message (which was written in a bit of a hurry): there is a clear distinction drawn by IUPAC at least between quantities which are unitless (or "dimensionless" in [their terminology](https://goldbook.iupac.org/terms/view/D01742)) and arbitrary units (or "procedure defined units" in [their terminology](https://en.wikipedia.org/wiki/Arbitrary_unit)).  The parent fraction clearly conforms to the former definition, and does not conform to the latter.

So I think that unitless is more appropriate here.  But can this be left as is for PET without having to amend the broader BIDS specification (at least until the issue has been more broadly discussed?)

Reply:
@mathesong sorry for the delay on this. Keeping track of individual threads in a PR of this size is... difficult.

>  Is it necessary to resolve the arbitrary vs unitless distinction for the broader BIDS specification now before the commenting period runs out for PET BIDS?

I think this can be resolved outside of the commenting period, but before the associated release. I would like to see a new issue specifically about the unit terminology, though, since it has a broad impact.

> But can this be left as is for PET without having to amend the broader BIDS specification (at least until the issue has been more broadly discussed?)

I'd prefer to use the same terminology across the specification. The other maintainers may have a different opinion on this though, and I wouldn't want to hold anything up if they don't agree. Pinging @bids-standard/maintainers.

Reply:
@tsalo : O, I completely understand!  Just finding this little discussion thread is hard enough as it is! :).  Thanks so much for getting back to me, and for starting up the issue (https://github.com/bids-standard/bids-specification/issues/731).  I'll follow this up in there then.